content/recipes/authoring/translation.md

```diff
@@ -7,4 +7,48 @@ next: /recipes/authoring/kumite/
+- **Avoid translations to languages which cannot support the idea of the kata.** For example, there's no use in translating a big integer arithmetic kata to Python, which supports arbitrary precision integers natively. Do not translate kata into languages where the difficulty would significantly differ: from high level languages to NASM or BF, or into languages where the kata becomes much easier because it can be solved with easy to use, advanced features of the standard library (or other available libraries).
+
+
+## Description
+
+- **Respect guidelines for [Writing a Kata Description](/recipes/authoring/kata-snippets/description/).**
+- **Use [Codewars Markdown Extensions](/references/markdown/extensions/#sequential-code-blocks)** to organize language specific parts of the description, if present. Add sections specific to your language if necessary.
+- If possible, **avoid displaying many language variants to the user**. Use conditional paragraphs to display information revelant to the currently viewed language.
+- If the language version can pass tests successfuly only when run with some specific language runtime, it should be specified in the description.
+
+
+## Code components
+
+- **Respect the [General Coding Guidelines](/recipes/authoring/kata-snippets/coding-general/), any conventions imposed by your [language](/languages/) and guidelines specific to each component of the kata**, e.g. the [preloaded](/recipes/authoring/kata-snippets/preloaded/) section, [reference solution](/recipes/authoring/kata-snippets/reference-solution/), [sample tests](/recipes/authoring/kata-snippets/sample-tests/) and [submission tests](/recipes/authoring/kata-snippets/full-tests/).
+- If possible, **sample tests and fixed tests should remain consistent between translations.** It may not always be feasible due to inherent differences in programming languages, but doing so can save follow-up questions on the language a user is attempting the kata in when a question on failing tests is raised in the kata discourse.
+- Sometimes, the description is not the best place to put some language specific information. For example, the translator may wish to add some information on arguments, data formats, memory management, etc. and keep the description language-agnostic at the same time. In such cases, it's allowed to put such information as an explanatory comment in the solution setup.
+
+
+## Review process
+
+- **The translation must be reviewed before it's approved.** If the reviewer is not familiar with the language of the translation, they should ask other users for help.
+- **Any issues found during review should be posted** as translation comments.
+- Unfortunately, the approval of a translation cannot be easily blocked by raising issues. If a translation has many issues, cannot be easily fixed, and there's justified concern that it could be approved prematurely while not keeping up to quality standards, **it should be rejected**.
```

Comment:
```suggestion
- If a translation has too many issues or cannot be easily fixed, and there's justified concern that it could be approved prematurely while not keeping up to quality standards, **it can be rejected**.
```

no need to emphasises the hole in the process x) => just state what can happen/be done.

note: "should" -> "can" because the current guidelines is fro the translator more than the reviewer

Reply:
I hoped them to be useful for both: aurhors while creating, and reviewers for verification. But yeah, wording usually suggests author.

Reply:
yes sure. But iirc, there is a "dedicated doc to be" about translations approval. That's why I suggests this here.

Reply:
The doc you mean somehow shifted and drifted and split into two parts. Quality guidelines (the ones you are reading) are meant to be applicable for authors, *and* for reviewers, as a kind of checklist they can use before approving new content.

I will review wording of all existing guidelines and try to balance it so they do not suggest authors as much, and take approves more into picture.

I was also thinking about another document, kind of manual, which would describe authoring and review process from functional side: views, editors, actions, options, etc. I have no good idea for this yet.